compiler/esy.json

```diff
@@ -60,6 +61,9 @@
     "ocaml": "4.12.0",
     "@opam/fp": "reasonml/reason-native:fp.opam#0ed854f986256e52e59aeecfa90e9af60f105b15",
     "@opam/fs": "reasonml/reason-native:fs.opam#0ed854f986256e52e59aeecfa90e9af60f105b15",
+    "@opam/ppx_deriving_yojson": "ocaml-ppx/ppx_deriving_yojson:ppx_deriving_yojson.opam#0809ea014a6cdd27d5863e05cf330ff74d0114ae",
+    "@opam/ppx_deriving_cmdliner": "hammerlab/ppx_deriving_cmdliner:ppx_deriving_cmdliner.opam#1f086651fe7f8dd98e371b09c6fcc4dbc6db1c7c",
+    "@opam/ocamlfind": "1.9.3",
```

Comment:
These resolutions are for the new, unreleased versions of the derivers for yojson and cmdliner that support the newer ppxlib that sedlex uses. (They may be published before this PR lands and can be removed.)